.clang-format

```diff
@@ -0,0 +1,110 @@
+# #############################################################################
+# clang-format file for Certifier Framework code base
+# #############################################################################
+Language:      Cpp
+BasedOnStyle:  Google
+
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+
+AlignConsecutiveAssignments: true
+AlignConsecutiveDeclarations: true
+AlignConsecutiveMacros: true
+AlignEscapedNewlines: Right
+
+AllowShortBlocksOnASingleLine: false
+AllowShortEnumsOnASingleLine: true
+AllowShortFunctionsOnASingleLine: Inline
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlignTrailingComments: true
+
+AlwaysBreakAfterReturnType: TopLevel
+AlwaysBreakBeforeMultilineStrings: false
+
+BinPackArguments: false
+BinPackParameters: false
+
+BreakBeforeBraces: Custom
+BraceWrapping:
+  AfterCaseLabel:  true
+  AfterClass:      false
+
+  # Multi-line if() and for() control statements will have their
+  # opening { come on a new line, for readability.
+  AfterControlStatement: MultiLine
+  AfterEnum:       false
+  AfterFunction:   true
+  AfterObjCDeclaration: false
+  AfterStruct:     false
+  AfterUnion:      false
+  AfterExternBlock: false
+  BeforeCatch:     false
+  BeforeElse:      false
+  IndentBraces:    false
+  SplitEmptyFunction: true
+  SplitEmptyRecord: false
+  SplitEmptyNamespace: false
+
+BreakStringLiterals: true
+
+BreakBeforeBinaryOperators: NonAssignment
+BreakBeforeTernaryOperators: true
+BreakStringLiterals: true
+
+ColumnLimit:     80
+CompactNamespaces: false
+ConstructorInitializerAllOnOneLineOrOnePerLine: true
+ConstructorInitializerIndentWidth: 3
+ContinuationIndentWidth: 4
+Cpp11BracedListStyle: true
+
+# Do not allow file-specific variations in pointer alignment.
+# Apply PointerAlignment consistently.
+DerivePointerAlignment: false
+
+FixNamespaceComments: true
+
+IndentCaseLabels: true
+IndentCaseBlocks: false
+IndentGotoLabels: false
+IndentPPDirectives: AfterHash
```

Comment:
@jlmucb : "I don't like "#    include <stdbool.h>"   again, wasted space for no readability benefit."

Reply:
Changing this from `IndentPPDirectives: AfterHash` to `IndentPPDirectives: None` (default), changes the output

from:

```
 41 #ifdef OE_CERTIFIER
 42 #  include "openenclave/attestation.h"
 43 #  include "openenclave/sealing.h"
 44 #endif
```

to:

```
 41 #ifdef OE_CERTIFIER
 42 #include "openenclave/attestation.h"
 43 #include "openenclave/sealing.h"
 44 #endif
```

Reply:
Recommendation: Go with John's preference: `IndentPPDirectives: None`

Reply:
Update: During final review, John agreed that it's ok to have nested `#include`'s and `#define`'s indented, for readability. Outermost #define and #includes will not have extra spaces in them.